bdb/bdb_schemachange.c

```diff
@@ -222,14 +222,25 @@ static int do_llog_int(bdb_state_type *ch_bdb_state, DBT *tbl, DBT *type,
 }
 
 static int do_llog(bdb_state_type *bdb_state, scdone_t sctype, char *tbl,
-                   int wait, int *bdberr)
+                   int wait, const char *origtable, int *bdberr)
 {
     DBT *dtbl = NULL;
     if (tbl) {
         dtbl = alloca(sizeof(DBT));
         bzero(dtbl, sizeof(DBT));
         dtbl->data = tbl;
         dtbl->size = strlen(tbl) + 1;
+        dtbl->size = strlen(bdb_state->name) + 1;
```

Comment:
Do we need this? Also, its overwriting the above assignment.

Reply:
typo, removed